tests/python/pants_test/util/test_objects.py

```diff
@@ -163,6 +163,23 @@ class MixedTyping(datatype([
   pass
 
 
+class SomeBaseClass(object):
+  @abstractmethod
+  def something(self): pass
+
+
+class SomeDatatypeClass(SomeBaseClass):
+  def something(self):
+    return 'asdf'
+
+  def __repr__(self):
+    return 'SomeDatatypeClass()'
+
+
+class WithSubclassTypeConstraint(datatype([('some_value', SubclassesOf(SomeBaseClass))])):
+  pass
```

Comment:
Pull up on previous line? That would fit in 100 cols and you place pass on the same line elsewhere, so the inconsistency draws the eye.

Reply:
Yes, agreed and fixed. I just realized I had deleted the file with directory-local variables in the pants repo setting my line length to 100 chars, so it was defaulting to 80 (and had been for a bit). Fixed that as well on my end.